src/Pos/Explorer/Web/Server.hs

```diff
@@ -156,6 +158,11 @@ explorerHandlers _sendActions =
 ----------------------------------------------------------------
 -- API Functions
 ----------------------------------------------------------------
+rights' :: [(Either a b, c)] -> [(b, c)]
```

Comment:
I don't really understand what this is for? Doesn't `rights` already take care of this behaviour:
```
rights   :: [Either a b] -> [b]
rights x = [a | Right a <- x]
```